### ISSUE TITLE
Auto deselect.

### ISSUE THREAD

**Issue by albertgh** [questbeat/QBImagePicker#33]
Hi, Thanks a lot for your code. 

I have a little suggest.

```objc
imagePickerController.allowsMultipleSelection = YES;
imagePickerController.minimumNumberOfSelection = 1;
imagePickerController.maximumNumberOfSelection = 1;
```
In this kind of case, when selecting a new picture can you make it deselect the old one automatically?
Then the user can select the new one without deselecting the old one before.
Because before tap the done button, user maybe change mind when scrolling.

**Comment by questbeat** [comment]
Implemented in [2.5.0](https://github.com/questbeat/QBImagePicker/tree/2.5.0).

**Comment by albertgh** [comment]
👍
